lib/squadster/formations/squad_member.ex

```diff
@@ -10,6 +10,7 @@ defmodule Squadster.Formations.SquadMember do
     field :role, RoleEnum
     belongs_to :user, Squadster.Accounts.User
     belongs_to :squad, Squadster.Formations.Squad
+    has_many :approved_squad_requests, Squadster.Formations.SquadRequest, foreign_key: :approver_id
```

Comment:
why there is `many`, correct me if i wrong, but one **member** can hold only one **approved_squad_request**

Reply:
maybe we should make `disapproved_squad_requests`, but in my o3.14nion :) it doesn't make any sence

Reply:
please, share you thoughts about this

Reply:
One member can hold multiple requests that was *approved by him*, e.g. commander/deputy can approve all the requests, and `approved_squad_requests` will return all of them. Not sure about disapproved, I think we can just delete request if it was declined. Also maybe this field isn't really needed, maybe we can leave only `approver` on request, but they don't load automatically and don't have any performance impact, so why not?

Reply:
Removed ✔️